torch/csrc/jit/lite_interpreter/mobile_module.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+//#include <ATen/core/function_schema.h>
+#include <torch/csrc/jit/lite_interpreter/bytecode.h>
+
+namespace torch{
+namespace jit{
+namespace mobile {
+using Stack = std::vector<c10::IValue>;
+
+class Function{
```

Comment:
Since there's run() in Function, put code related stuffs in Code struct (similar to JIT) and interpreter state will run the Code as a shared pointer.